.travis.yml

```diff
@@ -9,6 +9,7 @@ before_install:
   - go get github.com/mattn/goveralls
   - go get -u github.com/alecthomas/gometalinter
   - gometalinter --install
+  - go test -i $(go list ./... | grep -v /vendor | grep -v /tests)
```

Comment:
why adding this?

Reply:
gotype will cause could not import error.
according to the following PR, use go test -i to fix it

golang/go#12703
alecthomas/gometalinter#105

Reply:
comments like this should go into the file instead of staying in github, so that people will understand why 1 months later